frontend/src/js/actions/Actions.js

```diff
@@ -28,6 +28,13 @@ var Actions = {
             index: index
         }
     },
+    changePlot(value) {
+        return {
+            type: 'CHANGE_PLOT',
+            value,
+            value
```

Comment:
Is this supposed to be value twice? Also, what does this do? I'm not familiar with this syntax for objects.